ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml

```diff
@@ -893,25 +893,28 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-disk-encryption-set-minimal-permission
-- as: azure-ipi-disc-fullypriv-amd-mixarch-f28-destructive
-  cron: 35 4 28 * *
+- as: azure-ipi-disc-fullypriv-arm-mixarch-f2
```

Comment:
@jianlinliu are these changes on `azure-ipi-disc` on purpose in this PR?

Reply:
ah, I just re-order the two jobs, make no other change.

Reply:
ok, then let's just ignore it